drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1,5 +1,20 @@
 function DragAndDropTemplates(configuration) {
     "use strict";
+
+    var gettext;
+    if ('DragAndDropI18N' in window) {
+        // Use DnDv2's local translations
+        gettext = window.DragAndDropI18N.gettext;
+    } else if ('gettext' in window) {
+        // Use edxapp's global translations
+        gettext = window.gettext;
+    }
+    if (typeof gettext == "undefined") {
+        // No translations -- used by test environment
+        gettext = function gettext_stub(string) { return string; };
+    }
+    window.gettext = gettext;
```

Comment:
We can't do this -- it overrides the `gettext` for the rest of the javascript rendered on the Open edX page.